theatre/studio/src/panels/SequenceEditorPanel/RightOverlay/Marker.tsx

```diff
@@ -0,0 +1,83 @@
+import type {Pointer} from '@theatre/dataverse'
+import {useVal} from '@theatre/react'
+import {pointerEventsAutoInNormalMode} from '@theatre/studio/css'
+import getStudio from '@theatre/studio/getStudio'
+import {lockedCursorCssPropName} from '@theatre/studio/uiComponents/PointerEventsHandler'
+import useContextMenu from '@theatre/studio/uiComponents/simpleContextMenu/useContextMenu'
+import useRefAndState from '@theatre/studio/utils/useRefAndState'
+import React from 'react'
+import styled from 'styled-components'
+import {attributeNameThatLocksFramestamp} from '@theatre/studio/panels/SequenceEditorPanel/FrameStampPositionProvider'
+import type {SequenceEditorPanelLayout} from '@theatre/studio/panels/SequenceEditorPanel/layout/layout'
+
+const Mark = styled.div<{left: number}>`
+  position: absolute;
+  height: 20px;
+  width: 8px;
+  background-color: white;
+  z-index: 1;
+  color: black;
+  overflow: hidden;
+  font-size: 10px;
+  left: ${({left}) => left}px;
+
+  ${pointerEventsAutoInNormalMode};
+
+  #pointer-root.draggingPositionInSequenceEditor & {
+    pointer-events: auto;
+    cursor: var(${lockedCursorCssPropName});
+  }
```

Comment:
Question: What is the `&` for in the selector? I know how it works in `& div` in side a rule body, but not sure what it would do here.

Reply:
Added a comment 

```
// "All instances of this component inside #pointer-root when it has the .draggingPositionInSequenceEditor class"
  // ref: https://styled-components.com/docs/basics#pseudoelements-pseudoselectors-and-nesting
  ```